.github/workflows/ci.yml

```diff
@@ -67,3 +67,57 @@ jobs:
 
       - name: Unit Test
         run: yarn test
+
+  generate-swagger:
+    runs-on: ubuntu-latest
+    env:
+      SWAGGERHUB_API_KEY: ${{ secrets.SWAGGERHUB_API_KEY }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "dir=$(yarn config get cacheFolder)" >> $GITHUB_OUTPUT
+
+      - uses: actions/cache@v3
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Setup node and cache dependencies
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: "yarn"
+
+      - name: Install Dependencies
+        run: yarn install
+
+      - name: Nest Build
+        run: yarn build
+
+      - name: Install swaggerhub-cli
+        run: yarn global add swaggerhub-cli
+
+      - name: Read package version
+        uses: tyankatsu0105/read-package-version-actions@v1
+        id: package-version
+
+      - name: Generate Swagger JSON
+        run: yarn generate-swagger
+        env:
+          APP_SWAGGER_Version: ${{ steps.package-version.outputs.version }}
+
+      - name: Upload on Swaggerhub - Cophr_develop
+        if: github.ref != 'refs/heads/main'
+        run: |
+          swaggerhub api:update "Cophr/test/${{ env.APP_SWAGGER_Version }}" --setdefault --file=swagger-docs.json --visibility=public
```

Comment:
Use app version may easily result in conflict between PRs, may use commit hash or use other that may not easily to conflict to identify the version.

Reply:
I think use commit hash as the version is a good idea！

Reply:
In this commit is solved version by commit hash.
[refactor: swagger version by commit hash](https://github.com/Cophr/backend/pull/9/commits/aa357cea5759e832e4863f47b6d57075f0fb0c0f)
But in order to achieve automatic, use create instead of update.
[fix: not use update is use create](https://github.com/Cophr/backend/pull/9/commits/7d487a2219c18f65f66c0f7d91075d9da230554d)